### ISSUE TITLE
Apache2 install 

### ISSUE THREAD

**Issue by russell4** [grocy/grocy#328]
I did check out and try issue #121. 

Unfortunately, there is not an error to call out other than browsing to localhost/grocy/app.php or localhost/grocy/public/index.php pulls up a blank screen. Browsing to localhost/grocy/ brings up the directory listing in the browser.

I've tried changing BASE_URL to "/" and "/grocy" and "/grocy/"
When I run php -v, I see PHP 7.0.33-0+deb9u3 (cli) (built: Mar  8 2019 10:01:24) ( NTS )
SQLite ex1 -> SQLite version 3.16.2 2017-01-06 16:32:41

During reading through #121 I tried changing from the /var/www/html/grocy (original location) to /var/www/grocy , this still didn't work out with the same issues above.

Any thoughts on getting this running?

**Comment by berrnd** [comment]
You need at least PHP 7.1, I personally only test against PHP 7.2 currently - see #53, #8 and #166.

**Comment by russell4** [comment]
For conclusion's sake. I did the upgrade and getting a different error now. Since the original issue has been resolved, I feel like I should close this one, try debugging my new error before raising another issue. 

How I resolved the blank issue, php7.0 was indeed the culprit. I found a guide that walked through installing the 7.2 repo then after "a2dismod php7.0", "a2enmod php7.2", and "systemctl restart apache2" I was finally able to see php7.2 when I ran a phpinfo() page (for debugging).
